manifests/core-bases/base/httproute.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: odh-dashboard
+spec:
+  parentRefs:
+    - name: odh-gateway
```

Comment:
Looks like this is wrong as the default name odh operator uses is: https://github.com/opendatahub-io/opendatahub-operator/blob/4e2ab4b21afbb5b2d8c7df46b038456c36fbb690/internal/controller/services/gateway/gateway_support.go#L24 . Or am I missing something?

Also, I wonder whether this isn't something that should be configurable or at least - shouldn't we rather read this value from some odh/rhoai operator config (e.g. DSCI or DSC) or somewhere instead having this hardcoded? Or shall we rather search for it? Or are we truly okay to have this hardcoded?

Reply:
```suggestion
    - name: data-science-gateway
```